presto-memory-context/src/main/java/com/facebook/presto/memory/LocalMemoryContext.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.memory;
+
+import com.google.common.util.concurrent.ListenableFuture;
+
+public interface LocalMemoryContext
+{
+    long getBytes();
+    ListenableFuture<?> setBytes(long bytes);
+    boolean tryUpdateBytes(long bytes);
```

Comment:
The only caller of this method (`OperatorContext.trySetMemoryReservation`) actually wants a `trySetBytes`. Change this method to `trySetBytes` instead．